### ISSUE TITLE
False-positive LOCAL_VARIABLE_EARLY_DECLARATION

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#488]
## Describe the bug
```kotlin
    fun `method name incorrect, part 4`() {
        val code = """
                  class TestPackageName {
                    fun methODTREE(): String {
                    }
                  }
                """.trimIndent()
        lintMethod(code, LintError(2, 7, ruleId, "${FUNCTION_NAME_INCORRECT_CASE.warnText()} methODTREE", true))
    }
```

## Expected behavior
Variable `code` is declared right before it's first used, so there should be no warning.

## Observed behavior
```
Warn (2:2) [LOCAL_VARIABLE_EARLY_DECLARATION] local variables should be declared close to the line where they are first used: <code> is declared on line <2> and is used for the first time on line <8>
```

## Environment information
* diktat version: 0.1.3

**Comment by aktsay6** [comment]
Seems like this bug is not reproducable.
